frontend/package.json

```diff
@@ -10,7 +10,10 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@material-ui/core": "^4.12.4",
+    "@emotion/react": "^11.11.3",
+    "@emotion/styled": "^11.11.0",
```

Comment:
are we using emotion?

Reply:
Material UI requires it
https://mui.com/material-ui/getting-started/installation/#default-installation

Reply:
oh ok thanks for the link!